internal/suites/example/compose/redis/docker-compose.yml

```diff
@@ -1,8 +1,9 @@
 version: '3'
 services:
   redis:
```

Comment:
I do not believe this file will be required anymore if you're are using the redis-sentiel compose stack.

Reply:
I have merged them all into redis.